data/human/deep missions.txt

```diff
@@ -1072,7 +1072,7 @@ mission "Deep: Project Hawking"
 			label accept
 			`	"Thank you, Captain. The planets that we need to go to are..." Garrison turns to the other three scientists. "<stopovers>," Hannah says.`
 			`	"... Thanks. After we've gathered all the supplies, we'll need to be dropped off on <destination>."`
-			`	You bring the scientists to your ship where they quickly make themselves comfy in the bunk rooms. As you walk to the bridge, you overhear Pierre quietly communicating with what you assume to be the spaceport authorities, "... <last> agreed. I expect Project Hawking to proceed on schedule. We will arrive before <day>." As you prepare for launch, you wonder what you've just signed up for.`
+			`	You bring the scientists to your ship where they quickly make themselves comfy in the bunk rooms. As you walk to the bridge, you overhear Pierre quietly communicating with what you assume to be the spaceport authorities, "... <last> agreed. I expect Project Hawking to proceed on schedule. We will arrive before the <day>." As you prepare for launch, you wonder what you've just signed up for.`
```

Comment:
'the' is already included in <day>: https://github.com/endless-sky/endless-sky/wiki/CreatingMissions#text-replacements
```suggestion
			`	You bring the scientists to your ship where they quickly make themselves comfy in the bunk rooms. As you walk to the bridge, you overhear Pierre quietly communicating with what you assume to be the spaceport authorities, "... <last> agreed. I expect Project Hawking to proceed on schedule. We will arrive before <day>." As you prepare for launch, you wonder what you've just signed up for.`
```

Reply:
Looking at the code, it looks like the changes for date format types broke that and the string is no longer formed correctly. This text should remain as is, but the code needs to be corrected.

Reply:
> Looking at the code, it looks like the changes for date format types broke that and the string is no longer formed correctly. This text should remain as is, but the code needs to be corrected.

Do you know what the proper code should be?  if so, could you specify?

Reply:
> > Looking at the code, it looks like the changes for date format types broke that and the string is no longer formed correctly. This text should remain as is, but the code needs to be corrected.
> 
> Do you know what the proper code should be?  if so, could you specify?

If I knew what the code should be well enough to post it here, I could have just opened a PR with it, so no, I can't.

Reply:
😜 

Sorry for the misunderstood, I only made my comment because it sounded like you knew more about the codes (enough to know there is a better one) and had hoped that you could offer more clear direction. 

No offense intended.